### PR TITLE
[Style] clean up code

### PR DESCRIPTION
**Pull request by greatSumini**
## Description

rename divider, titleTextVertical
remove lint warnings
add types
add snapshots

## Related Issues

none

## Tests

I added the following tests:

none

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Card/index.tsx [hunk 1]
```diff
@@ -124,21 +123,17 @@ const Card: FC<Props> = (props) => {
...
       {renderTitle && (
-        <TitleContainer
-          style={[titleContainerStyle]}
-          titleTextVertical={titleVertical}>
+        <TitleContainer style={[titleContainerStyle]} hasSubTitle={!!subTitle}>
           <TitleText style={[titleStyle]}> {title} </TitleText>
-          {subTitle && subTitle.length > 0 ? (
+          {subTitle ? (
             <SubTitleText style={[subTitleStyle]}> {subTitle} </SubTitleText>
           ) : null}
```
Comment (sk97kang) [inline main/Card/index.tsx]: I want to give unity with other parts.

```jsx
{subTitle &&  <SubTitleText style={[subTitleStyle]}> {subTitle} </SubTitleText>}
```

File: stories/dooboo-ui/Card.stories.tsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ function Default(): React.ReactElement {
         }}
         title={'This is title'}
         titleStyle={{ display: 'flex', flexDirection: 'column' }}>
-        <Text>I don't have Image</Text>
+        <Text>{"I don't have Image"}</Text>
```
Comment (sk97kang) [inline stories/dooboo-ui/Card.stories.tsx:92]: Just question : Why use '{ }' ?

File: main/Card/index.tsx (hunk 1 above)
Comment (greatSumini) [reply main/Card/index.tsx]: 👌🏻

File: stories/dooboo-ui/Card.stories.tsx (hunk 2 above)
Comment (greatSumini) [reply stories/dooboo-ui/Card.stories.tsx:92]: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md
to remove this eslint error

Comment (ghost) [reply stories/dooboo-ui/Card.stories.tsx:92]: Ok.!!

File: main/Card/index.tsx (hunk 1 above)
Comment (greatSumini) [reply main/Card/index.tsx]: 105bc0a

**Pull request merged**